src/hacbs/components/PipelineRunDetailsView/PipelineRunDetailsView.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { useK8sWatchResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { Bullseye, Spinner } from '@patternfly/react-core';
+import { NamespaceContext } from '../../../components/NamespacedPage/NamespacedPage';
+import { PipelineRunGroupVersionKind } from '../../models';
+import { PipelineRunKind } from '../../types';
+import { calculateDuration } from '../../utils/pipeline-utils';
+import './PipelineRunDetailsView.scss';
+
+export interface PipelineRunDetailsProps {
+  pipelineRunName: string;
+}
+
+export const PipelineRunsDetailsView: React.FC<PipelineRunDetailsProps> = ({ pipelineRunName }) => {
```

Comment:
Component name should match file name.
```suggestion
export const PipelineRunDetailsView: React.FC<PipelineRunDetailsProps> = ({ pipelineRunName }) => {
```

Reply:
matched these for all i saw there was a mismatch